AppServicesUsageSamples/apps/presence-detection/README.md

```diff
@@ -0,0 +1,55 @@
+# Presence detection
+
+User presence detection allows to check for connectivity or activity of user or device. It has many
+applications from showing what users are available in a chat application to identify what devices are 
+malfunctioning in an industrial process.
+
+This sample shows how to implement user presence detection effectively using Realm and Atlas resources.
+
+## Listening for local connectivity
+
+Clients could detect whether they are connected to Atlas or not via the `SyncSession.connectionState` 
+property. The values of the connection state are `DISCONNECTED`, `CONNECTING`, or `CONNECTED`.
+
+This property support flows via the function `connectionStateAsFlow`, if simplifies listening to 
+connectivity changes:
+
+```kotlin
+realm.syncSession
+    .connectionStateAsFlow()
+    .collect {connectionStateChange ->
+        // handle connection state
+    }
+```
+
+In this sample we use local connectivity detection to change the behavior of the connectivity control
+button, it would disconnect or connect the session based on whether there is or not connectivity.
```

Comment:
A bit unclear what this means. Users might now have any idea about the buttons inside. Better to describe behavior in general terms so people know if it matches their use case.

Reply:
I have changed to more general terms.